### PR TITLE
Blood Cult removals and tweaks

### PR DESCRIPTION
**Pull request by Robustin**
My philosophy for blood cult was always "easy to learn, hard to master". The first part got most of the emphasis last year as I gut the useless and redundant runes and talismans and streamlined most of the mechanics. 

I plan on adding new cult content, including runes and talismans, but until then I feel some housekeeping is in order because I put a pretty strict limit on making sure you can view all runes and talismans without having to scroll. 

I'm reverting Joan's reversion of my removal of the Drain Life rune. It is a consistently unused that does nothing but clutter and confuse. 

I'm also removing the fake runes talisman. I don't think I've ever seen it used and the new cult mechanics (tomes appearing on convertees, animated runes, the inevitable blood/tomes/gibs that tend to accumulate in any base where you could create a fake-rune talisman) make it all but useless. 

The veiling talisman and the shackling talisman have gained more charges before they are depleted. 

"Raise Dead" is now renamed "Resurrect Cultist". Considering theres another rune that actually brings ghosts to life, the name clarity was needed.

"Astral Communion" is now "Spirit Sight". We get 1% more flavor from obtuse names but far less usability, see: Clock Cult.

### CODE REVIEW

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -850,7 +850,7 @@ var/list/wall_runes = list()
 	light_color = LIGHT_COLOR_LAVA
 	req_cultists = 3
 	construct_invoke = 0
-	var/tick_damage = 25
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/runes.dm]: The total damage is 150, not 75, because it does both brute AND burn

Comment (Robustin) [inline code/game/gamemodes/cult/runes.dm]: fucking

Comment (ChangelingRain) [inline code/game/gamemodes/cult/runes.dm]: a better combat rune? no shit you do, blood boil is absurdly lethal.

**Comment by Supermichael777** [comment]
Its been months old man wheres my summoning nerf?

**Comment by KorPhaeron** [comment]
I dont think changing names we have used for 7 years makes it less confusing

"Raise Dead" is pretty straight forward.

**Comment by uraniummeltdown** [comment]
Don't change the names, we shouldn't pander to the illiterate
Also why remove Drain Life? That rune is damn handy for healing up from manifests

**Comment by Robustin** [comment]
I'm adamant that names should tell you what they do at a glance. 

Raise dead is one of the best cult runes, yet it gets scribed almost never. Personally even after years of playing cult I would still get the triumvirate of raise dead --> astral journey/communion ---> manifest spirit mixed up. Is raise dead the one that turns ghosts into people? Manifest spirit sounds like what I would use to enter ghost form... right? Astral Journey? Uh is that for teleporting? 

Go and look at the old stats. Cults win rate has practically doubled from its old numbers despite eating significant nerfs. Nar-Sie runes used to be scribed in 3 seconds and invoked without any warning FFS!

Making the mode more approachable has done wonders. If the tome had a more advanced interface I might contemplate giving runes more enigmatic names but when its clunky enough as it is people should be able to see what things do in a snap. 

Raise dead is just one example, the name wasn't a riddle but there were enough nuances and overlap with other runes that caused frustration for newer players. I've seen far too many people trying to resurrect non-cultists, or try to use it as a manifest rune.

@Supermichael777 

The balance/feature stuff is coming, these are just QOL and housekeeping.

**Comment by uraniummeltdown** [comment]
Why remove Drain Life though? It's a useful rune

**Comment by coiax** [comment]
But disguising runes is good for ones in public areas...

**Comment by Robustin** [comment]
Uranium, its simply not useful enough to justify it takes up a rune spot. We have pylons and unholy water for healing/rapid healing. Cults also have uses for almost any body, so dragging someone ripe for conversion or sac and going through a convoluted draining ritual just never serves a serious purpose. 

Out of the 1000's of runes that get scribed every month, drain life shows up like 15 times and its usually out of curiosity before the scriber goes "oh, that seems pretty useless". 

First off Coiax its a talisman. Second, I created it, I wanted it to be a viable tactic, but I learned that with the current mechanics its just not feasible. It just encourages a high risk tactic that never works out because of the inherent tells in cult activity. Its probably only decent for disguising teleport runes but even that tended to have sec yelling about runes and by the time someone with a brain shows up to claim its fake, a cultist teleports in and confirms the opposite. 

So basically its never used, and when it does get used it defeats the purpose of a secret cult by revealing the cult early through slip-ups in the ruse.

**Comment by coiax** [comment]
Did this get merged with no changelog? ...
